sorc/hafs_tools.fd/sorc/hafs_obs_preproc/CMakeLists.txt

```diff
@@ -87,7 +87,7 @@ target_link_options(${exe_name} PRIVATE
 
 target_link_libraries(
   ${exe_name}
-  bufr::bufr_d
+  bufr::bufr_4
```

Comment:
This is always curious to me and I did not get time to check this. Are we sure bufr_d is exactly the same as bufr_4?

Reply:
No. If memory serves:
bufr_d = 8 byte integers, 4 byte reals
bufr_4 = 4 byte integers, 4 byte reals
bufr_8 = 8 byte integers, 8 byte reals

The latest versions of bufr only know how to build bufr_4

Reply:
Thanks, @SamuelTrahanNOAA ! (I think you mean the "integers" and "reals" to be swapped)

I took a little bit of time to dig into this change. It looks like they changed to use a Fortran/C interface to consolidate _4, _d and _8 versions into one library (but renamed as _4, which is unexpected to me).
https://github.com/NOAA-EMC/NCEPLIBS-bufr/issues/78